docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     ports:
       - "${CLIENT_SERVICE_PORT}:3000"
     container_name: landgriffon-client
-    command: develop
+    command: start
```

Comment:
This does not work (at least for local dev) because NextJS need a previous build (generates a `build` folder in the FS by `next build` and then runs it by `next start`.

`landgriffon-client | Error: Could not find a production build in the '/opt/landgriffon-client/.next' directory. Try building your app with 'next build' before starting the production server.`

This is the reason I intended to add a [single-command build&run](https://github.com/Vizzuality/landgriffon/pull/24#discussion_r649328722) but got rejected

Reply:
I changed this back to develop.  I am also changing both entrypoint.sh files to more closely resemble each other, and to make them more explicit in their sub-commands, and thus require less brain power to manage. (this will go against the change I requested before, I know, sorry about it, but humans do change their minds sometimes).

The docker-compose is used exclusively for local dev, the only requirement is that it plays nice with the Dockerfile's we have for both apps - those are used to build the images that go to prod. If you want to modify this file to better suit your local dev env, be my guest.